optimum/commands/export/neuron.py

```diff
@@ -65,8 +65,8 @@ def parse_args_neuron(parser: "ArgumentParser"):
     )
     optional_group.add_argument(
         "--auto_cast_type",
-        default=None,
-        help='The data type to cast FP32 operations to when auto-cast mode is enabled. Can be `"bf16"`, `"fp16"` or `"tf32"`.',
+        default="bf16",
```

Comment:
Maybe add a choices argument to make sure it is one of the 4 values.

Reply:
Done!